indexing-service/src/test/java/org/apache/druid/indexing/appenderator/BatchAppenderatorDriverTest.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.druid.segment.realtime.appenderator;
+package org.apache.druid.indexing.appenderator;
```

Comment:
Class `BatchAppenderatorTester` that must be moved also if we want to move the test classes to the `server` module is very difficult to move because it uses the constructor for class `IndexTask.IndexTuningConfig` which is not available in the server module. Moving that class will demand moving a lot of classes that do not make sense in the `server` module.

Reply:
I am excluding the class again.. I see no easy way to moving them all to the same package. We are fighting against a tool (coverage) primarily, secondarily against module design (i.e. "server" has some batch stuff and it should only have "realtime" stuff). Resolving the latter, which takes non-trivial effort and should go in another ticket, will also resolve the former.

Reply:
Removed class from exclusion after finding a way to move the test classes to the server module.